test/stable/common/job.rb

```diff
@@ -3,15 +3,15 @@
 
 class Test < ChartTest
```

Comment:
We should rename this file to `job_spec.rb`, otherwise it won't be picked up by the test framework. Should have caught that in the initial PR, sorry about that :(

Reply:
fixed.